api-reference/50 Common/Object Structures/positionConfig/at/at.md

```diff
@@ -5,26 +5,113 @@ type: String | Object
 ---
 ---
 ##### shortDescription
-The target element position that the widget is positioned against.
+Specifies a side or corner of the [target element](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) at which the overlay element should be positioned. 
```

Comment:
Change to:
Specifies the [target element's](/Documentation/ApiReference/Common/Object_Structures/positionConfig/#of) side or corner where the overlay element should be positioned.